paddle/fluid/inference/api/analysis_predictor.cc

```diff
@@ -192,9 +192,7 @@ void AnalysisPredictor::SetMkldnnThreadID(int tid) {
 bool AnalysisPredictor::Run(const std::vector<PaddleTensor> &inputs,
                             std::vector<PaddleTensor> *output_data,
                             int batch_size) {
-  if (UNLIKELY(config_.cpu_math_library_num_threads() > 1)) {
-    paddle::platform::SetNumThreads(config_.cpu_math_library_num_threads());
-  }
+  paddle::platform::SetNumThreads(config_.cpu_math_library_num_threads());
```

Comment:
Here the reason of remove `UNLIKELY(config_.cpu_math_library_num_threads() > 1)` is: https://github.com/PaddlePaddle/Paddle/issues/17153#issuecomment-489487740
I test this PR in pyramid_dnn, it doesn't impact the inference speedup in small model. @Superjomn

Reply:
omp_set_num_threads() just does nothing but set var, so the cost should be less.